launcher/game/EasyDialogsWin.py

```diff
@@ -673,10 +673,10 @@ def hookProc(hdlg, uiMsg, wParam, lParam):
     ofn.lpfnHook = LPOFNHOOKPROC(hookProc)
 
     if fn(ctypes.byref(ofn)):
-        filenames = filter(None, filename.split('\0'))
+        filenames = [fn for fn in filename.split('\0') if fn]
         if len(filenames) > 1:
             dir, filenames = filenames[0], filenames[1:]
-            return map(lambda fn: os.path.join(dir, fn), filenames)
+            return [os.path.join(dir, fn) for fn in filenames]

```

Comment:
The same thing here. Reusing the identifier `fn` when it is used as a function earlier is confusing.

Reply:
I realize the original code also reused the same `fn` name but it seems to me renaming it something else like `pn` (for "path name") would be better.

Reply:
Thanks. Updated. Not sure if we should change that in this PR though.